modules/nlu/src/backend/engine.ts

```diff
@@ -9,6 +9,7 @@ import SlotTagger from './slots/slot-tagger'
 import { isPatternValid } from './tools/patterns-utils'
 import { computeKmeans, ProcessIntents, Trainer, TrainInput, TrainOutput } from './training-pipeline'
 import { EntityCacheDump, ListEntity, ListEntityModel, NLUEngine, Tools, TrainingSession } from './typings'
+import elect from './legacy-election'
```

Comment:
can you call this electIntent or anything more specify than elect

maybe electionProcess or electionPipeline or maybe legacyElectionPipeline